QgisModelBaker/utils/ili2db_utils.py

```diff
@@ -0,0 +1,99 @@
+"""
+/***************************************************************************
+                              -------------------
+        begin                : 27.09.2024
+        git sha              : :%H$
+        copyright            : (C) 2024 by Germán Carrillo
+        email                : german at opengis ch
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.core import Qgis, QgsMessageLog
+from qgis.PyQt.QtCore import QObject, Qt, pyqtSignal
+
+from QgisModelBaker.libs.modelbaker.iliwrapper import ilideleter
+from QgisModelBaker.libs.modelbaker.iliwrapper.ili2dbconfig import (
+    DeleteConfiguration,
+    Ili2DbCommandConfiguration,
+)
+from QgisModelBaker.libs.modelbaker.iliwrapper.ili2dbutils import JavaNotFoundError
+from QgisModelBaker.libs.modelbaker.utils.qt_utils import OverrideCursor
+
+
+class Ili2DbUtils(QObject):
```

Comment:
Why's that not in the Library?

Reply:
I'll move it :)

Reply:
Done in https://github.com/opengisch/QgisModelBaker/pull/955/commits/805ee0e52f71ebdef39596a17800a77d9a5f641e and https://github.com/opengisch/QgisModelBakerLibrary/pull/107/commits/db232bd899618b02e79fb15595c3300aa2c6b90f